components/contribute-cards/EditTierModal.js

```diff
@@ -2,6 +2,7 @@
 /* eslint-disable react/prop-types */
```

Comment:
I missed that this file was in `components/contribute-cards`, I don't think it's the best place for it.
Suggested location: `components/edit-collective/tiers/EditTierModal.js`